addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5519,6 +5523,10 @@ registry.ImageTools = ImageHandlerOption.extend({
         if (params.optionsPossibleValues.resetCrop) {
             return this._isCropped();
         }
+        // Only the 'Crop' widget is visible when image is not supported for style options.
+        if (_.intersection(_.keys(params.optionsPossibleValues), ['transform', 'selectStyle']).length) {
```

Comment:
It seems to work because `_computeWidgetVisibility` is called for **"all sub-widgets with main widgets params"** [on `SnippetOptionWidget` > `updateUIVisibility()`] which means for `crop`, `resetCrop`, `transform` and `resetTransform` (no sub-widgets), the visibility will be checked with their methods params so `params.optionsPossibleValues.resetCrop != undefined` means we are checking the 'resetCrop' visibility.